packages/block-library/src/page-list/block.json

```diff
@@ -25,7 +25,12 @@
 	],
 	"supports": {
 		"reusable": false,
-		"html": false
+		"html": false,
+		"color": {
+			"background": true,
+			"link": true,
+			"text": true
```

Comment:
Page list only ever has links (plus bullet points) so making "link" option available means we can end up with a different colour for the bullet points and for the links:
<img width="625" alt="Screen Shot 2021-11-19 at 10 41 02 am" src="https://user-images.githubusercontent.com/8096000/142514228-9223d8c3-447b-41f9-9443-15afd542123d.png">

Which, granted, someone _might_ want to do, but is likely only going to cause confusion. Having only background and text color settings should be enough.

Reply:
> Which, granted, someone might want to do, but is likely only going to cause confusion

Agreed -- this is a good point 👍 

> Having only background and text color settings should be enough.

Do you happen to mean background + link color settings? I imagine the user would be more worried about changing the color of the text than the bullet point.

Reply:
The text color setting will change both links and bullets; the link color will only change the links. If I were setting a color on the block, I would expect it to change everything; not sure there's a use case for changing just the links.